build/gulp/tasks/perf.ts

```diff
@@ -99,10 +99,13 @@ const createMarkdownTable = (
 
   return markdownTable([
     ['Example', ...fieldLabels],
-    ..._.map(exampleMeasures, (exampleMeasure, exampleName) => [
-      exampleName,
-      ...fieldValues[exampleName],
-    ]),
+    ..._.sortBy(
+      _.map(exampleMeasures, (exampleMeasure, exampleName) => [
+        exampleName,
+        ...fieldValues[exampleName],
+      ]),
+      row => -row[fields.indexOf('median')],
```

Comment:
I've sorted the `yarn perf` results by their median values with the slowest components up top.